.storybook/documentation.scss

```diff
@@ -41,3 +41,13 @@
 
 @import './src/recipes/domains-sidebar.scss';
 @import './src/recipes/16six-sidebar.scss';
+
```

Comment:
@Jeremy-Walton These will be removed in a followup PR correct?

Reply:
This is just in the documentation and not packaged in the exported styles. It's purely to show card examples.